### PR TITLE
migrations(plugins): move Slack and PD to new_sentry_plugins

### PR DESCRIPTION
**Pull request by scefali**
Must be merged with: https://github.com/getsentry/sentry-plugins/pull/546

### CODE REVIEW

File: src/test_only_plugins/anonymizeip.py
Comment (joshuarli) [inline src/test_only_plugins/anonymizeip.py]: Can you bring this file back, under `new_sentry_plugins`? Looks like it's used by `src/new_sentry_plugins/splunk/plugin.py`.

**Review by wedamija — APPROVED** [review]
This looks good. Let's test this on staging before we deploy, since these are important plugins.

Should we also stop installing `sentry-plugins` in getsentry as part of this, or do you want to do it afterwards?

**Comment by scefali** [comment]
@wedamija Already put this on staging but I can put it on staging after merging master as well. Your call. Might be smart to play around with it now so we can get a second pair of eyes on it.

I want to stop installing `sentry-plugins` in getsentry after this PR.

**Comment by wedamija** [comment]
> @wedamija Already put this on staging but I can put it on staging after merging master as well. Your call. Might be smart to play around with it now so we can get a second pair of eyes on it.
> 
> I want to stop installing `sentry-plugins` in getsentry after this PR.

Let's just put it up on staging after we merge everything real quick and just confirm that they're still installed, it'll only take an extra 5-10 minutes and just gives us some extra peace of mind.

**Review by joshuarli — APPROVED** [review]

**Pull request merged**
